src/tactic/command/global_search_trigger.py

```diff
@@ -86,6 +86,12 @@ def execute(my):
         data.update( my.cleanup(caller.get_value("description", no_exception=True) ))
         data.update( my.cleanup(caller.get_value("keywords", no_exception=True) ))
 
+        # Updates for adding changed keywords into asset keywords_data column
+        if sobj.column_exists("keywords_data"):
+            user_keywords = input.get("update_data").get("keywords")
+            if user_keywords not in ['none', None]:
```

Comment:
why would user_keywords be 'none' at times?, should eliminate that case on the js side